AMM/project/contracts/AMM-contract/tests/utils/mod.rs

```diff
@@ -9,13 +9,21 @@ abigen!(
     "./project/contracts/exchange-contract/out/debug/exchange-contract-abi.json"
 );
 
+pub const HARDCODED_EXCHANGE_CONTRACT_BYTECODE_ROOT: &str =
```

Comment:
Why does this need to state that it's hardcoded? It doesn't convey anything meaningful imo and it's long

Reply:
![1](https://user-images.githubusercontent.com/103777923/204401942-59350488-b98b-423e-921b-cfe5b04c3493.jpeg)

Reply:
Yes.